### ISSUE TITLE
Router: type is not defined

### ISSUE THREAD

**Issue by davidbrooksio** [cferdinandi/reef#105]
Hi

This might be linked to my other issue [https://github.com/cferdinandi/reef/issues/69](https://github.com/cferdinandi/reef/issues/69)

Version 8.0 seemed to fix the issues with the router but I'm afraid v8.2 seems to have introduced a different bug. Again, I am using webpack (5.21.2) but when I package the bundle I get the following console error:
```
Uncaught ReferenceError: type is not defined
    at x.addComponent (router.js:2)
    at new x (reef.es.min.js:2)
    at Module../src/docs/index.js (index.js:59)
    at __webpack_require__ (bootstrap:18)
    at startup:3
    at startup:5
```
This is referencing file node_modules/reefjs/router.js on line 2.

It's not really applicable to produce a reduced test case but I am happy to create a repo if it will help.

**Comment by davidbrooksio** [comment]
I have updated my repo at [https://github.com/davidbrooksio/reefsinglepage](https://github.com/davidbrooksio/reefsinglepage) to make things a little easier.

**Comment by cferdinandi** [comment]
Yep, I see the error on my end here. I’ll push a fix for this this morning as soon as I’m back at my computer.

**Comment by cferdinandi** [comment]
Fixed in v8.2.1

**Comment by davidbrooksio** [comment]
The fix works for me, Thank you :)
